consensus/src/extension/election.rs

```diff
@@ -356,4 +357,31 @@ mod test {
             }
         }
     }
+
+    #[test]
+    #[ignore]
```

Comment:
Maybe slap a `// TODO(A0-XXXX): this test should pass` above the `#[ignore]`. It's extremely unlikely you'll forget, but just two days ago such a comment reminded me about leftover code, so I'm particularly fond of them.